droid-configs.inc

```diff
@@ -514,7 +514,7 @@ fi
 %post
 # Force an update of ssu repositories
 touch %{board_mapping_dir}/*
-touch %{_datadir}/ssu/features.d/*
+touch %{_datadir}/ssu/features.d/* || :
```

Comment:
Depends on your shell?
Info: Installing: droid-config-$DEVICE .....
/bin/touch: cannot touch `/usr/share/ssu/features.d/*': No such file or
directory

How likely the mechanisms in RPM are to change?